codec_specs.md

```diff
@@ -783,3 +783,22 @@ Codec Name: VobBtn Buttons
 
 Description: Based on [MPEG/VOB PCI packets](http://dvd.sourceforge.net/dvdinfo/pci_pkt.html). The file contains a header consisting of the string "butonDVD" followed by the width and height in pixels (16 bits integer each) and 4 reserved bytes. The rest is full [PCI packets](http://dvd.sourceforge.net/dvdinfo/pci_pkt.html).
 
+## Block Addition Mappings
+
+Registered `BlockAddIDType` are:
+
+### Use BlockAddIDValue
+
+Block type identifier: 0
+
+Block type name: Use BlockAddIDValue
+
+Description: This value indicates that `BlockAddIDValue` should be used as `BlockAddIDType`.
```

Comment:
I find this wording to be somewhat confusing (same in the xml). I'd prefer something more like

> This value indicates that the actual type is stored in `BlockAddIDValue` instead.

and maybe some more words mentioning that this is due to compatibility with legacy files or some such.

Reply:
>and maybe some more words mentioning that this is due to compatibility with legacy files

Indeed, I strongly second this.